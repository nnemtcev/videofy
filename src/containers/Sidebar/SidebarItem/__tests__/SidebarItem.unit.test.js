import { shallow } from "enzyme";
import React from "react";

import { SidebarItem } from "../SidebarItem.js";

describe("SidebarItem", () => {
  test("SidebarItem renders with no props passed in", () => {
    const wrapper = shallow(<SidebarItem />);
    expect(wrapper).toMatchSnapshot();
  });

  test("highlighted SidebarItem renders with icon and label", () => {
    const wrapper = shallow(
      <SidebarItem highlighted icon="fire" label="Trending" />
    );
    expect(wrapper).toMatchSnapshot();
  });

  test("non-highlighted SidebarItem renders with icon and label", () => {
    const wrapper = shallow(<SidebarItem icon="fire" label="Trending" />);
    expect(wrapper).toMatchSnapshot();
  });
});
