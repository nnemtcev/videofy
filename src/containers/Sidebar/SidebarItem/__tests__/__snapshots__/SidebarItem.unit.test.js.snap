// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarItem SidebarItem renders with no props passed in 1`] = `
<MenuItem
  className="sidebar-item "
>
  <div
    className="sidebar-item-alignment-container"
  >
    <span>
      <Icon
        as="i"
        size="large"
      />
       
    </span>
    <span />
  </div>
</MenuItem>
`;

exports[`SidebarItem highlighted SidebarItem renders with icon and label 1`] = `
<MenuItem
  className="sidebar-item "
>
  <div
    className="sidebar-item-alignment-container"
  >
    <span>
      <Icon
        as="i"
        name="fire"
        size="large"
      />
       
    </span>
    <span>
      Trending
    </span>
  </div>
</MenuItem>
`;

exports[`SidebarItem non-highlighted SidebarItem renders with icon and label 1`] = `
<MenuItem
  className="sidebar-item "
>
  <div
    className="sidebar-item-alignment-container"
  >
    <span>
      <Icon
        as="i"
        name="fire"
        size="large"
      />
       
    </span>
    <span>
      Trending
    </span>
  </div>
</MenuItem>
`;
